manifests/p/Patchdeck/Agent/2.5.0/Patchdeck.Agent.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+﻿# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-19041-2364.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Patchdeck.Agent
+PackageVersion: 2.5.0
+PackageLocale: en-US
+Publisher: Patchdeck LLC
+# PublisherUrl: 
+# PublisherSupportUrl: 
+PrivacyUrl: https://patchdeck.com/privacy-policy
+# Author: 
+PackageName: Patchdeck Windows Agent
```

```suggestion
PackageName: PatchdeckWindowsAgent
```
This should match Control Panel